packages/block-library/src/navigation-link/index.php

```diff
@@ -146,7 +146,7 @@ function render_block_core_navigation_link( $attributes, $content, $block ) {
 			'style' => $style_attribute,
 		)
 	);
-	$html               = '<li ' . $wrapper_attributes . '>' .
+	$html               = '<div ' . $wrapper_attributes . '>' .
```

Comment:
Following this line of thinking, I think it should be possible to drop this div wrapper too here in navigation-link and home-link.

This change would need to rework the css for how submenus work, of course.

This is non blocking since changes here work well and now markup validates properly, but I think it'd be worth exploring.

| Example before | Example after |
|-----|-----|
| <img width="930" alt="Screen Shot 2021-05-14 at 10 10 14 AM" src="https://user-images.githubusercontent.com/1270189/118306964-eb460880-b49e-11eb-8911-faadcff9025f.png"> | <img width="745" alt="Screen Shot 2021-05-14 at 10 24 46 AM" src="https://user-images.githubusercontent.com/1270189/118306935-e2553700-b49e-11eb-8ac7-5701eca1e55a.png"> |

Reply:
Definitely, but I didn't want to touch the styling as part of this PR. :)

Reply:
Yeah, this could be done as a follow up too if we choose this approach 👍